langlib/langlib-test/src/test/resources/test-src/stringlib_test.bal

```diff
@@ -687,6 +688,138 @@ function testIncludesMatch() {
     assertTrue(result6);
 }
 
+function testFromBytesAsync() {
+    foreach int i in 0 ... 4 {
```

Comment:
Shall we increase the iterations to 10 and reduce the sleep to value like 0.002?